src/index.ts

```diff
@@ -24,6 +25,19 @@ const _isConnectedListeners = new Map<
   Types.NetInfoChangeHandler
 >();
 
+/**
+ * Sets configuration for NetInfo
+ */
+export function setNetInfoConfig(
+  newConfig: Partial<Types.NetInfoConfig>,
+): void {
+  Object.assign(netInfoConfig, newConfig);
+}
+
+export function getNetInfoConfig(): Types.NetInfoConfig {
+  return netInfoConfig;
+}
+
```

Comment:
These new methods should be added to the default exports at the bottom of this file.